zeppelin-zengine/src/main/java/org/apache/zeppelin/storage/LocalConfigStorage.java

```diff
@@ -128,7 +128,8 @@ static void atomicWriteToFile(String content, File file) throws IOException {
     }
     try {
       file.getParentFile().mkdirs();
-      Files.move(tempFile.toPath(), destinationFilePath,  StandardCopyOption.ATOMIC_MOVE);
+      Files.move(tempFile.toPath(), destinationFilePath,
+              StandardCopyOption.REPLACE_EXISTING); //StandardCopyOption.ATOMIC_MOVE);
```

Comment:
Why have this changed?

Reply:
Atomic move caused errors on Linux, I suggest asking khud about that